### ISSUE TITLE
[Architect v1.0.3] - Firmware Upgrade Notes

### ISSUE THREAD

**Issue by architectbugs** [BlackstarDigitalTeam/KetteringReleases#255]
Architect: v1.0.3
OS: Windows 10
Amp Model: Silverline Deluxe
Amp Version: 1.6.0
Description: Could FW upgrade notes be provided please? Thanks

**Comment by RichHaynes8** [comment]
Yes, will have to get Neil/Kartikeya to write them before putting them in manually
